schema.sql

```diff
@@ -8,4 +8,8 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 )
-ALTER TABLE animals ADD species VARCHAR(255);
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR(255);
+CREATE TABLE owners(id SERIAL PRIMARY KEY, full_name VARCHAR, age INT);
+CREATE TABLE species(id SERIAL PRIMARY KEY, name VARCHAR);
+ALTER TABLE animals ADD owner_id INT;
```

Comment:
- [ ] Please add a foreign key to the owner table.
     - You can consider using this code `ALTER TABLE animals ADD owner_id int REFERENCES owners(id);`